Facepunch.Steamworks/Interfaces/Workshop.cs

```diff
@@ -123,6 +123,37 @@ public Item GetItem( ulong itemid )
             return new Item( itemid, this );
         }
 
+        /// <summary>
+        /// The amount of item's the user is subscribed to for this App
+        /// </summary>
+        public uint GetAmountSubsribed()
```

Comment:
Spelling, should also probably be a property (although we could argue that there should just be a property of subscribed items, and you could just do Workshop.Subscriptions.Length)